README.md

```diff
@@ -1006,6 +1006,18 @@ in inheritance.
     end
     ```
 
+* When defining ActiveRecord model scopes, wrap the relation in a `lambda`.  A
+  naked relation forces a database connection to be established at class load
+  time (instance startup).
+
+    ```ruby
+    # bad
+    scope :foo, where(:bar => 1)
```

Comment:
I assume this is the case for this syntax as well:
```rb
scope :active, :conditions => ["deleted_at IS NULL"]
```
Let's maybe add this example as well, since we have it a lot in the codebase.